qpsolvers/solvers/__init__.py

```diff
@@ -706,6 +706,48 @@
 except ImportError:
     pass
 
+# qpax
+# ========
+
+qpax_solve_problem: Optional[
+    Callable[
+        [
+            Problem,
+            Optional[ndarray],
+            bool,
+        ],
+        Solution,
+    ]
+] = None
+
+qpax_solve_qp: Optional[
+    Callable[
+        [
+            Union[ndarray, csc_matrix],
+            ndarray,
+            Optional[Union[ndarray, csc_matrix]],
+            Optional[ndarray],
+            Optional[Union[ndarray, csc_matrix]],
+            Optional[ndarray],
+            Optional[ndarray],
+            Optional[ndarray],
+            Optional[ndarray],
+            bool,
+        ],
+        Optional[ndarray],
+    ]
+] = None
+
+try:
+    from .qpax_ import qpax_solve_problem, qpax_solve_qp
+
+    solve_function["qpax"] = qpax_solve_problem
+    available_solvers.append("qpax")
+    dense_solvers.append("qpax")
+    sparse_solvers.append("qpax")
+except ImportError as e:
+    raise e
```

Comment:
From the PyPI CI workflow:

```

Run python -c "import qpsolvers"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/runner/work/qpsolvers/qpsolvers/qpsolvers/__init__.py", line 9, in <module>
    from ._internals import available_solvers
  File "/home/runner/work/qpsolvers/qpsolvers/qpsolvers/_internals.py", line 11, in <module>
    from .solvers import available_solvers as supported_solvers
  File "/home/runner/work/qpsolvers/qpsolvers/qpsolvers/solvers/__init__.py", line 749, in <module>
    raise e
  File "/home/runner/work/qpsolvers/qpsolvers/qpsolvers/solvers/__init__.py", line 742, in <module>
    from .qpax_ import qpax_solve_problem, qpax_solve_qp
  File "/home/runner/work/qpsolvers/qpsolvers/qpsolvers/solvers/qpax_.py", line 17, in <module>
    import qpax
ModuleNotFoundError: No module named 'qpax'
```

Here the change would be to simply pass, as we do with other not-found solvers:

```suggestion
    pass
```